friendly_shipping.gemspec

```diff
@@ -24,21 +24,21 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "data_uri", "~> 0.0.3"
   spec.add_runtime_dependency "dry-monads", "~> 1.0"
-  spec.add_runtime_dependency "money", ">= 6.0.0"
-  spec.add_runtime_dependency "nokogiri", ">= 1.6"
-  spec.add_runtime_dependency "physical", ">= 0.4.4"
+  spec.add_runtime_dependency "money", "~> 6.0", ">= 6.0.0"
```

Comment:
The second version does not seem to have any effect, since `~> 6.0` implies `>= 6.0.0`